symfony/src/Twig/AppExtensions.php

```diff
@@ -108,6 +109,15 @@ public function isNew(Creator $creator): bool
         return NewArtisan::isNew($creator);
     }
 
+    public function isMinor(Creator $creator): bool
+    {
+        return match ($creator->getAges()) {
+            Ages::MINORS => true,
+            Ages::MIXED => true,
+            default => false,
+        };
```

Comment:
This should be sufficient if this is agreed on in https://github.com/veelkoov/fuzzrake/pull/244#issuecomment-2366154233

```suggestion
        return $creator->getAges() === Ages::MINORS;
```